ddtrace/tracer/tracer.go

```diff
@@ -314,34 +283,22 @@ func (t *tracer) Extract(carrier interface{}) (ddtrace.SpanContext, error) {
 }
 
 // flushTraces will push any currently buffered traces to the server.
```

Comment:
Comment update after renaming the method.

Reply:
👍